README.md

```diff
@@ -1,3 +1,26 @@
+<<<<<<< HEAD
```

Comment:
This shows the start of the conflict that git has found, if you try to see this file in rich diff, you will see that the header of the readme is duplicated. You will need to delete one of them.